CommonEnemy/src/main/java/dk/sdu/se/f23/InVasion/commonenemy/Enemy.java

```diff
@@ -1,52 +1,33 @@
 package dk.sdu.se.f23.InVasion.commonenemy;
+
 import dk.sdu.se.f23.InVasion.common.data.Entity;
 import dk.sdu.se.f23.InVasion.common.data.Point;
-import java.util.ArrayList;
+
 import java.util.Collections;
 import java.util.List;
 
 public class Enemy extends Entity {
-    private List<Point> route = new ArrayList<>();
-    private float speed;
+    private final List<Point> route;
```

Comment:
```suggestion
    private final List<Point> ROUTE;
```

Reply:
Route is not a constant since the points within the list can change